hack/local-up-cluster.sh

```diff
@@ -110,18 +110,6 @@ START_MODE=${START_MODE:-"all"}
 # A list of controllers to enable
 KUBE_CONTROLLERS="${KUBE_CONTROLLERS:-"*"}"
 
-# sanity check for OpenStack provider
-if [ "${CLOUD_PROVIDER}" == "openstack" ]; then
-    if [ "${CLOUD_CONFIG}" == "" ]; then
```

Comment:
you can use default config here

Reply:
Right, we don't need to do extra validation for just OpenStack.